authorization/src/main/kotlin/org/modelix/authorization/AuthorizationPlugin.kt

```diff
@@ -132,6 +162,7 @@ object ModelixAuthorization : BaseRouteScopedPlugin<IModelixAuthorizationConfig,
                                 buildPermissionPage(call.getPermissionEvaluator())
                             }
                         }
+                        installPermissionManagementHandlers()
```

Comment:
This is only enabled when `config.debugEndpointsEnabled` is `true`.

Reply:
Added a separate config option `permissionManagementEnabled` for that.